bitcoinspv/config/config.go

```diff
@@ -0,0 +1,86 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/btcsuite/btcd/btcutil"
+	"github.com/spf13/viper"
+	"go.uber.org/zap"
+)
+
+var (
+	defaultAppDataDir = btcutil.AppDataDir("native-bitcoin-spv", false)
```

Comment:
change the names

Reply:
Updateed names, split variables to make it cleaner